docs/getting-started.md

```diff
@@ -134,6 +134,14 @@ This command will ask you a series of questions in order to initialize your new
 
 And that’s it! Your new project is ready to go. 🎉
 
+<div class="tip" label="Info">
```

Comment:
```suggestion
<div class="note">
```

I’d rather not introduce a fourth category of note here; I use `note` for “additional technical details” type notes.